tavern/tomes/process_list/main.eldritch

```diff
@@ -30,11 +43,14 @@ def process_list(cmd_substring):
             current_proc_ppid = str(proc['ppid'])
             current_proc_username = proc['username']
 
-            print(pad_pid(current_proc_pid))
-            print(pad_pid(current_proc_ppid))
-            print(pad_username(current_proc_username))
-            print(current_proc_command.replace("\n","\\n")+"\n")
+            rows.append([
+                str(proc['pid']),
+                str(proc['ppid']),
+                proc['username'],
+                current_proc_command.replace("\n","\\n")[:80] # trim absurdly long cmds
```

Comment:
I'd rather not trim the command - in tavern we don't wrap so it won't look gross.
But if we avoid truncating that's more searchable fields.
Good for situations like REALLY long java commands with a "PASSWORD" arg.